client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -42,8 +43,8 @@ export default function Header( { isEditorSidebarOpened, toggleGeneralSidebar }:
 				role="toolbar"
 			></div>
 			<div className="gutenboarding__header-actions">
-				<Button isPrimary isLarge>
-					{ NO__( 'Continue' ) }
+				<Button isPrimary isLarge disabled={ isEmpty( siteVertical ) }>
```

Comment:
The button shows up for me as disabled even after selecting a vertical -- this is because we don't have `siteVertical` in state yet. It might be a bit confusing to use it here prior to actually introducing it.